superset-frontend/src/explore/components/controls/VizTypeControl/VizTypeGallery.tsx

```diff
@@ -469,83 +578,147 @@ export default function VizTypeGallery(props: VizTypeGalleryProps) {
         onChange(null);
       }
     },
     [
       stopSearching,
       isSearchFocused,
-      activeCategory,
+      activeSelector,
       selectedVizMetadata,
       onChange,
     ],
   );
 
+  const clearSelector = useCallback(e => {
+    e.stopPropagation();
+    if (isSearchFocused) {
+      stopSearching();
+    }
+    // clear current selector and set all charts
+    setActiveSelector(ALL_CHARTS);
+  }, []);
+
   const vizEntriesToDisplay = isActivelySearching
     ? searchResults
-    : chartsByCategory[activeCategory] || [];
+    : activeSelector === ALL_CHARTS
+    ? sortedMetadata
+    : chartsByCategory[activeSelector] || chartsByTags[activeSelector] || [];
 
   return (
     <VizPickerLayout className={className}>
       <LeftPane>
-        <SearchWrapper>
-          <Input
-            type="text"
-            ref={searchInputRef as any /* cast required because emotion */}
-            value={searchInputValue}
-            placeholder={t('Search')}
-            onChange={changeSearch}
-            onFocus={focusSearch}
-            data-test={`${VIZ_TYPE_CONTROL_TEST_ID}__search-input`}
-            prefix={
-              <InputIconAlignment>
-                <Icons.Search iconSize="m" />
-              </InputIconAlignment>
-            }
-            suffix={
-              <InputIconAlignment>
-                {searchInputValue && (
-                  <Icons.XLarge iconSize="m" onClick={stopSearching} />
-                )}
-              </InputIconAlignment>
-            }
-          />
-        </SearchWrapper>
-        <CategoriesWrapper>
-          {categories.map(category => (
-            <CategorySelector
-              key={category}
-              category={category}
-              isSelected={!isActivelySearching && category === activeCategory}
-              onClick={selectCategory}
-            />
-          ))}
-        </CategoriesWrapper>
+        <Wrapper>
+          <Collapse
+            expandIconPosition="right"
+            ghost
+            defaultActiveKey={Object.values(COLLAPSE_LABEL)}
+          >
+            <Collapse.Panel
+              header={
+                <span className="header">
+                  {COLLAPSE_LABEL.RECOMMENDED_TAGS}
+                </span>
+              }
+              key={COLLAPSE_LABEL.RECOMMENDED_TAGS}
+            >
+              {RECOMMENDED_TAGS.map(tag => (
+                <Selector
+                  key={tag}
+                  selector={tag}
+                  type="Tags"
+                  isSelected={!isActivelySearching && tag === activeSelector}
+                  onClick={clickSelector}
+                  onClear={clearSelector}
+                />
+              ))}
+            </Collapse.Panel>
+            <Collapse.Panel
+              header={
+                <span className="header">{COLLAPSE_LABEL.ALL_CHARTS}</span>
+              }
+              key={COLLAPSE_LABEL.ALL_CHARTS}
+            >
+              <Selector
```

Comment:
It's a bit strange to have a collapse panel that will only ever have one item in it. Should we put the "All Charts" selector at the top of the left pane instead, outside of the collapse?